ansible/roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
I believe it's better to install this role to `$HOME/.ansible/roles` instead of doing it in the current project directory and checking it into Git.
But up to you, of course — maybe you want to pin exact version of this role and modify something inside it.